layers/meta-resin-qemu/conf/samples/local.conf.sample

```diff
@@ -67,6 +67,5 @@ BB_DISKMON_DIRS = "\
 # libsdl library available on your build system.
 PACKAGECONFIG_append_pn-qemu-native = " sdl"
 PACKAGECONFIG_append_pn-nativesdk-qemu = " sdl"
-ASSUME_PROVIDED += "libsdl-native"
```

Comment:
I think there was a problem with the native built of this library. Was this fixed in the later versions of poky?